app/views/settings/dex_connector_ldaps/_form.html.slim

```diff
@@ -11,11 +13,18 @@
     = f.label :host
     = f.text_field :host, class: "form-control", value: @certificate_holder.host
     = error_messages_for(@certificate_holder, :host)
+    small.form-text.text-muted id="#{@certificate_holder.host}_ldap_host_help"
+      | Host name of LDAP server reachable from the cluster
 
   .form-group class="#{error_class_for(@certificate_holder, :port)}"
     = f.label :port
-    = f.text_field :port, class: "form-control", value: @certificate_holder.port
```

Comment:
You can revert this since the user can go back and forth through the pages and they wouldn't be seeing what they submitted previously.

Reply:
reverted.